src/services/Libs.ts

```diff
@@ -403,8 +428,7 @@ export const throwErrorNotification = (e: Error) => {
 /**
  * Trim leading and ending dot of a string
  */
-export const trimDot = (str: string) => str.replace(/^[\.]+|[\.]+$/g, '');
-
+export const trimDot = (str: string) => str.replace(/^[.]+|[.]+$/g, '');
```

Comment:
Regular Expression.  Keep Backslashes for escapes.  Otherwise `.` means any one character.

Reply:
See Above.